src/Sylius/Bundle/WebBundle/Resources/views/Frontend/Checkout/Step/thankYou.html.twig

```diff
@@ -0,0 +1,27 @@
+{% extends 'SyliusWebBundle:Frontend:layout.html.twig' %}
+
+{% set taxAdjustment = constant('Sylius\\Component\\Core\\Model\\AdjustmentInterface::TAX_ADJUSTMENT') %}
+{% set shippingAdjustment = constant('Sylius\\Component\\Core\\Model\\AdjustmentInterface::SHIPPING_ADJUSTMENT') %}
+{% set promotionAdjustment = constant('Sylius\\Component\\Core\\Model\\AdjustmentInterface::PROMOTION_ADJUSTMENT') %}
+
+{% block content %}
+<div class="page-header">
+    {% if order.state == 'confirmed' %}
+        <h1>{{ 'sylius.checkout.thank_you.header'|trans|raw }}</h1>
+    {% else %}
+        <h1>{{ 'sylius.checkout.thank_you.header'|trans|raw }}</h1>
+    {% endif %}
+</div>
+
+<div class="row">
+    <div class="col-md-12">
+        <h4>Thank you for your order</h4>
```

Comment:
I agree, this was WIP, but I stopped regarding @pjedrzejewski comments above. If he already has this completed then I think we should wait